_observing-your-data/prometheusmetrics.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: Prometheus SQL and Metrics Analytics Observability plugin
+nav_order: 40
+redirect_from:
+  - /observing-your-data/prometheusmetrics/
+---
+
+# Prometheus SQL and Metrics Analytics Observability plugin
+Introduced 2.4
+{: .label .label-purple }
+
+Starting with OpenSearch 2.4, you can use two new additional features within the **Observability** application in OpenSearch Dashboards: the Metrics UI for visualization, and connecting to Prometheus to gather metrics.
+
+![Prometheus and Metrics]({{site.url}}{{site.baseurl}}/images/metrics/metricsgif.gif)
+
+## Metrics Analytics UI
+
+The Metrics Analytics UI is a new component in the **Observability** application for OpenSearch Dashboards, as seen in the following image.
+
+![Metrics UI example 1]({{site.url}}{{site.baseurl}}/images/metrics/metrics1.png)
+
+You can view visualizations from created metrics in the **Metrics analytics** section of the **Observability** application as seen in the following image.
+
+![Metrics UI example 2]({{site.url}}{{site.baseurl}}/images/metrics/metrics2.png)
+
+Additionally, you can save a visualization created with [Event analytics]({{site.url}}{{site.baseurl}}/observing-your-data/event-analytics/) as seen in the following image.
+
+Only queries that include a Time Series visualization and stats/span can be saved as a metric.
+{: .note }
+
+![Metrics UI example 3]({{site.url}}{{site.baseurl}}/images/metrics/metrics3.png)
+
+## Prometheus connection
```

Comment:
"Creating a connection to Prometheus" - maybe this should be the first section?

Reply:
yeah I'm cool with that